rdr_service/tools/tool_libs/codes_management.py

```diff
@@ -216,12 +221,64 @@ def run(self, skip_file_export_write=False):
             if not self.args.dry_run:
                 if exit_code == 1:
                     session.rollback()
-                elif not skip_file_export_write:
+                elif 'prod' in self.gcp_env.project:
+                    # We only want to write the export file for prod
                     self.write_export_file(session)
 
         return exit_code
 
 
+class CodeHierarchyMap:
+    """Constructs a data structure for accessing the parent and module code values for a given code id"""
+
+    def __init__(self, session: Session):
+        # Load all surveys with the question and option codes used
+        query = session.query(
```

Comment:
:nit should we move this to the CodeDao?

Reply:
Good idea, that would make it more reusable. I'll work on that.

Reply:
Ok, it's moved over